plugins/mtd/mtd.quirk

```diff
@@ -16,3 +16,21 @@ Flags = no-probe
 [MTD\NAME_BIOS]
 Name = Internal SPI Controller
 #FirmwareGType = FuIfdFirmware
+
+# AMD Kria System on Module
+[MTD\VENDOR_xlnx&PRODUCT_ZynqMP-KV260-revB&NAME_Image-A-FSBL,-PMU,-ATF,-U-Boot]
```

Comment:
I think we want comma in the strsafe list too -- we use that in some places to split the instance-ids.